lib/customers.rb

```diff
@@ -0,0 +1,12 @@
+class Customers
+    attr_accessor :a_customer
+    
+    def initialize
+        @a_customer = {}
+    end
+
+    def create_customer(customer_name, mobile_number)
+        a_customer[customer_name] = mobile_number
```

Comment:
I'm not sure what this code means - does it set the mobile_number as the value, and the customer_name is the key?